draft-ietf-quic-recovery.md

```diff
@@ -131,8 +132,9 @@ mechanisms ensure that data and frames that need reliable delivery are
 acknowledged or declared lost and sent in new packets as necessary. The types
 of frames contained in a packet affect recovery and congestion control logic:
 
-* All packets are acknowledged, though packets that contain only ACK
-  and PADDING frames are not acknowledged immediately.
+* All packets are acknowledged, though packets that contain no
+  retransmittable frames are only acknowledged along with retransmittable
+  packets.
```

Comment:
How about: "Packets containing only ACK or PADDING frames are not required to be acknowledged immediately by a receiver. They might be eventually included in acknowledgements when subsequent packets containing retransmittable frames are received by the receiver."

Reply:
I like what I have in this PR currently better.  Is there something you don't like about the current text that you're trying to fix?